src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,32 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
what about constant naming? You can see example [here](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)